core/client/src/main/java/alluxio/client/file/FileSystemUtils.java

```diff
@@ -156,7 +156,9 @@ public Boolean apply(Void input) {
         try {
           return fs.getStatus(uri).isPersisted();
         } catch (Exception e) {
-          throw Throwables.propagate(e);
+          Throwables.propagateIfPossible(e);
+          throw new RuntimeException(e);
+          //throw Throwables.propagate(e);
```

Comment:
I change my code as fllow:
```java
Throwables.throwIfUnchecked(e);
throw new RuntimeException(e);
```
But, when I compile whith maven as "mvn clean install", there have a error said that couldn't find the throwIfUnchecked function. What should I do?